packages/cactus-test-tooling/src/main/typescript/corda/corda-test-ledger.ts

```diff
@@ -270,9 +270,11 @@ export class CordaTestLedger implements ITestLedger {
         hasDbMigrations: false,
       });
       this.log.debug(`Pulled jar (%o bytes) %o`, jarPath, jar.length);
+    } catch (e) {
```

Comment:
@petermetz I put this catch because I didn't want to throw an exception, each example has a different filename but I wanted it to go to the next one in case it couldn't find it. Now it is a switch case